### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by qsefthuopq**

### CODE REVIEW

File: src/main/resources/assets/identity/lang/zh_cn.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {
   "identity.unlock_entity": "已解锁%s种化身",
   "identity.revoke_entity": "你的对%s化身的记忆已化为尘埃..."
+   "identity.grant_success": "成功将化身%s给予%s。",
```
Comment (AllanChain) [inline src/main/resources/assets/identity/lang/zh_cn.json]: Invalid syntax here. You forgot to append comma to line 3.
```suggestion
  "identity.revoke_entity": "你的对%s化身的记忆已化为尘埃...",
  "identity.grant_success": "成功将化身%s给予%s。",
```

**Comment by James103** [comment]
@qsefthuopq Have you tried force-pushing? That can collapse multiple small but related commits into one larger commit.

**Comment by AllanChain** [comment]
@James103 No more violence. Just squash merge.

**Comment by Draylar** [comment]
Thanks!

**Pull request merged**